types/wordpress__notices/index.d.ts

```diff
@@ -1,10 +1,10 @@
-// Type definitions for @wordpress/notices 1.5
+// Type definitions for @wordpress/notices 3.3
```

Comment:
This is my only doubt with this PR. Was this simply updated to the latest package version, or did this actually change from 3.2 to 3.3?

(I find [unpkg](https://unpkg.com/browse/@wordpress/notices@1.5.0/) helpful to answer this type of question)

Reply:
> Was this simply updated to the latest package version, or did this actually change from 3.2 to 3.3?

I simply updated it to the latest package version. Looks like `onclick` and others were changed/added in different versions. I think [`2.12.0`](https://unpkg.com/browse/@wordpress/notices@2.12.0/src/store/actions.js) is the closest version of these changes.

Do you think we should change the version number to it?